doc/man1/openssl-s_server.pod.in

```diff
@@ -727,6 +682,11 @@ by the client in binary mode.
 
 {- $OpenSSL::safe::opt_r_item -}
 
+{- $OpenSSL::safe::opt_v_item -}
+
+Verification errors are display, for debugging, but the command will
```

Comment:
Ditto, but could mention that this applies only when client certificates are solicited by server and provided by the client.

Reply:
Put "If the server requests a client certificate, than ..." at the start of the sentence.  Fixed the typo.